src/guide/1/essentials/installing-kuzzle/index.md

```diff
@@ -0,0 +1,378 @@
+---
+layout: full.html.hbs
+algolia: true
+title: Install Kuzzle
+order: 1
+---
+
+# Installing Kuzzle
+
+In this section we'll describe different ways of installing Kuzzle.
+
+---
+
+## Automated script
+
+The easiest way to install Kuzzle is by using our installation script:
+
+```bash
+bash -c "$(curl http://get.kuzzle.io)"
+```
+
+---
+
+## Docker
+
+Before launching Kuzzle using Docker containers, ensure that your system meets the following requirements:
+
+- **64-bit environment**
+- **Docker v1.10+**, see [instructions here](https://docs.docker.com/engine/installation/)
+- **Docker Compose v1.8+**, see [instructions here](https://docs.docker.com/compose/install/)
+
+To install docker, you need to download the docker-compose file:
+
+
+Before starting the docker stack, you need to increase the maximum amount of virtual memory in order to run Elasticsearch, which is part of our stack (see why <a href="https://www.elastic.co/guide/1/en/elasticsearch/reference/5.x/vm-max-map-count.html">here</a>):
+
+```bash
+sudo sysctl -w vm.max_map_count=262144
+```
+
+To make this configuration permanent, you need to update your `/etc/sysctl.conf` file:
+
+```bash
+echo "vm.max_map_count=262144" >> /etc/sysctl.conf
+```
+
+Now, we can start the docker stack:
+
+```bash
+mkdir kuzzle-docker
+cd kuzzle-docker
+wget http://kuzzle.io/docker-compose.yml
+docker-compose up
+```
+
+Your terminal should now be logging startup messages from the Kuzzle stack. After a few seconds, you should see the following message:
+
+```bash
+# kuzzle_1         | [✔] Kuzzle server ready
+```
+
+Your Kuzzle is now up and running. For a quick test, you can explore the main HTTP API endpoint by clicking this link <a href="http://localhost:7512">http://localhost:7512</a> or by using cURL on the command line:
+
+```bash
+curl "http://localhost:7512?pretty"
+```
+
+---
```

Comment:
Should we mention the ability to launch a stack that supports HTTPS? Maybe a simple link to the corresponding section in the docs should be enough...

Reply:
do we have that?